src/Action/Generate.hs

```diff
@@ -201,17 +201,26 @@ actionGenerate g@Generate{..} = withTiming (if debug then Just $ replaceExtensio
     createDirectoryIfMissing True $ takeDirectory database
     whenLoud $ putStrLn $ "Generating files to " ++ takeDirectory database
 
+    let warnFlagIgnored thisFlag reason ignoredFlagPred ignoredFlag =
+          when ignoredFlagPred $ putStrLn $ "Warning: " <> thisFlag <> " is " <> reason <> ", which means " <> ignoredFlag <> " is ignored."
```

Comment:
Everywhere else in this file uses ++ for string concat

Reply:
Ah, sorry, the power of habit

Reply:
No big deal - not worth fussing about either way